lc2fen/infer_pieces.py

```diff
@@ -222,7 +222,11 @@ def infer_chess_pieces(pieces_probs, a1_pos, previous_fen=None):
         if (
             max_pieces_left[max_idx] > 0
             and out_preds[square] is None
-            and __check_bishop(max_idx, tops, w_bishop_sq, b_bishop_sq)
+            # and __check_bishop(max_idx, tops, w_bishop_sq, b_bishop_sq)
```

Comment:
Good catch!

Usually it isn't good practice to leave commented code in a version-controlled repository (since you can go back in the versions to check old code). Could you move this comment to the documentation of the `__check_bishop` function? I think it would still be a good idea to leave the `__check_bishop` in the code because it could be useful for future optimizations.

Reply:
Good idea! I just did that. I wasn't sure whether to comment out the entire `__check_bishop()` function or not--I left it uncommented even though the function isn't called anywhere in the code